src/main/kotlin/org/malachite/estella/commons/Payload.kt

```diff
@@ -0,0 +1,17 @@
+package org.malachite.estella.commons
+
+import org.malachite.estella.interview.domain.InvalidUUIDException
+import java.util.*
+
+
+data class PayloadUUID(val uuid: String) {
+    fun toUUID(): UUID {
+        try {
+            return UUID.fromString(uuid)
+        } catch (ex: IllegalArgumentException) {
+            ex.printStackTrace()
+            throw InvalidUUIDException()
```

Comment:
printStackTrace popieram, ale nie byłoby lepiej, żeby zwracać `UUID?` i uniknąć tego throwa?

Reply:
Znaczy nie wiem czy jest sens to obsługiwać bo korzystamy z tego głównie RequestParamsach więc jak nie podasz złe UUID to i tak z tym nic nie zrobimy chyba sensowniejszego

Reply:
Ogólnie popieram Radka. Zwłaszcza że jeśli UUID będzie złe to ani z tym nic nie zrobimy, a dodatkowo wiemy, że to nie sprawka frontu, bo tam bezpośrednio nikt uuid nie podaje. Ponadto z frontu może przyjść undefined i też ciula będziesz wiedział jak to obsłużyć, więc koniec końców I tak będziesz musiał rzucić jakiś exception (na przykład not found). Tu będziemy wiedzieli o tym wcześniej.
Cytując klasyka "rzucaj wcześnie, łap późno"